server/services/bookerService.ts

```diff
@@ -3,10 +3,10 @@ import { HmppsAuthClient, OrchestrationApiClient, RestClientBuilder } from '../d
 import { AuthDetailDto, PrisonerInfoDto, VisitorInfoDto } from '../data/orchestrationApiTypes'
 
 export interface Prisoner extends PrisonerInfoDto {
-  displayId: number
+  prisonerDisplayId: number
 }
 export interface Visitor extends VisitorInfoDto {
-  displayId: number
+  visitorDisplayId: number
```

Comment:
This is the crux of the change: be explicit in naming what kind of `displayId` it is. Everything else is updating to reflect this change.